### ISSUE TITLE
Error on MacOS with undefined: unix.SYS_IOCTL

### ISSUE THREAD

**Issue by nandub** [la5nta/pat#227]
I'm currently getting the error when `go get ..`
```
go get github.com/la5nta/pat
# github.com/albenik/go-serial
/usr/local/lib/go/src/github.com/albenik/go-serial/zsyscall_darwin.go:16:27: undefined: unix.SYS_IOCTL
```

MacOS High Sierra - 10.13.6 (17G14033)
go version go1.14.5 darwin/amd64

**Comment by martinhpedersen** [comment]
Thanks. This seems to be the same problem as described here: https://github.com/la5nta/pat/pull/222#issuecomment-705051033.

The PR should fix it, but I have not had the time to review it properly yet :/

**Comment by martinhpedersen** [comment]
Fixed in 103f239

**Comment by nandub** [comment]
@martinhpedersen thanks.
